index.html

```diff
@@ -6,36 +6,26 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="./assets/css/styles.css">
-    <title>First Project</title>
+    <title>Javier Martinez's Portfolio</title>
 </head>
 
 <body>
     <header>
-        <div class="topHeader">
-            <div class="logo">
-                <a href="#" rel="noopener" target="_blank">
-                    My Logo
-                </a>
-            </div>
-            <nav>
-                <ul class="links-index">
-                    <li>
-                        <a href="#">Portfolio</a>
-                    </li>
-                    <li>
-                        <a href="#">About</a>
-                    </li>
-                    <li>
-                        <a href="#">Contact</a>
-                    </li>
-                </ul>
-                <div id="breadcrumb">
-                    <a href="#" rel="noopener" target="_blank">
-                        <img src="./assets/images/breadcrumb.png" alt="">
-                    </a>
-                </div>
-            </nav>
+        <div class="logo">
+          <a href="#">My Logo</a>
         </div>
+  
+        <nav>
+          <div class="hamburger">
+            <img src="./assets/images/breadcrumb.png" alt="" />
+          </div>
+          <div class="main-menu">
```

Comment:
- [ ] [OPTIONAL] I advise you to make the HTML scroll behavior smooth, this will make it look more pleasing and feel more enticing 😀.

```
html {
  scroll-behavior: smooth;
}
```

Reply:
![05](https://user-images.githubusercontent.com/86327867/231979310-25f7a3cb-628a-4910-8ee0-3021684af215.jpg)

Applying scroll-behavior as optional suggestion.